From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@siriusk-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Could you correct the "intermmediate" spelling mistake in this pull request change? Thanks

vigneshcmsft committed:
Update stream-analytics-out-of-order-and-late-events.md

-Changed documentation to be explicit about late arrival and out of order window. Removed text that didn't add any additional semantics.
-Added example.

itechedit committed:
edit pass: notification-hubs-aspnet-cross-platform-notification

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into notification-hubs-aspnet-cross-platform-notification

itechedit committed:
edit pass: notification-hubs-aspnet-cross-platform-notification

itechedit committed:
edit pass: notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

itechedit committed:
edit pass: notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

itechedit committed:
edit pass: notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

itechedit committed:
Update notification-hubs-aspnet-cross-platform-notification.md

itechedit committed:
Update notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns.md

itechedit committed:
edit pass: notification-hubs-windows-store-dotnet-get-started-wns-push-notification

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into notification-hubs-windows-store-dotnet-get-started-wns-push-notification

itechedit committed:
edit pass: notification-hubs-windows-store-dotnet-get-started-wns-push-notification

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into notification-hubs-windows-store-dotnet-get-started-wns-push-notification

itechedit committed:
edit pass: notification-hubs-windows-store-dotnet-get-started-wns-push-notification

itechedit committed:
Update notification-hubs-portal-create-new-hub.md

itechedit committed:
Update notification-hubs-portal-create-new-hub.md

itechedit committed:
Update notification-hubs-windows-store-dotnet-get-started-wns-push-notification.md

itechedit committed:
Update notification-hubs-windows-store-dotnet-get-started-wns-push-notification.md

itechedit committed:
edit pass: includes-notification-hubs-aspnet-backend-notifyusers

itechedit committed:
Update notification-hubs-aspnet-backend-notifyusers.md

itechedit committed:
Update notification-hubs-aspnet-backend-notifyusers.md

Commit:
task 401: consolodate CDN quota info

Commit:
task 377 - update POP locations for Akamai and reorg table

Commit:
story 401: consolodate CDN quota info - fix table in cdn-limits.md

vigneshcmsft committed:
Addressing comments

-	Fixed comments by Kati in PR.
-	Changed the ordering, using the same font.
-	Yes, for event 2, it should be 00:00:01. Changed.
-	Added comments for events 2 and 3.

vigneshcmsft committed:
Addressing comments

Adding #sign-off to complete pull request

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-1708

Commit:
task 316: organize Verizon and Akamai compression info into 2 separate notes, Acrolynx edits

Commit:
story 403, update 4 screen shot to match current screens, Acrolynx edits

v-thepet committed:
Merge branch 'release-mvc-iot-dps' of https://github.com/Microsoft/azure-docs-pr into fixlinks

Commit:
task 315 - add validation status table

curtand committed:
Update 1 article from classic to Azure portal

paulathurman committed:
edit pass: azure-security-threat-modeling-tool-feature-overview

paulathurman committed:
edit pass: azure-security-threat-modeling-tool-feature-overview

anirudhcavale committed:
Reflecting most up to date changes

TerryLanfear committed:
update to FAQ

curtand committed:
revise per PR review

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-dev-guide-updates

sethmanheim committed:
Adding article about migrating from ACS to SAS

rmca14 committed:
Merge pull request #21610 from curtand/class0907

Update 1 article from classic to Azure portal

Commit:
missed AMQP reference

sethmanheim committed:
Acrolinx

tdykstra committed:
Clarify "Window scripting"

sethmanheim committed:
Adding article to TOC

vigneshcmsft committed:
Fixed few editorial issues.

Commit:
Merge pull request #21731 from tdykstra/patch-1

Clarify "Window scripting"

Commit:
forced push error

Commit:
ca04

sethmanheim committed:
Adding a link

Commit:
validation remnant

ahmedelnably committed:
Fixing Command

PRMerger18 committed:
Merge pull request #21735 from MarkusVi/ca04

ca04

Commit:
minor edit

Commit:
Merge pull request #21736 from ahmedelnably/master

Fixing Command

Commit:
minor edit

vigneshcmsft committed:
Fixed an example

#sign-off

Commit:
ca05

TerryLanfear committed:
final fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sec-acs

PRMerger19 committed:
Merge pull request #21371 from msebolt/iot-hub-amqp-update-revised

removal of MQTT restrictive clause

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sec-acs

vigneshcmsft committed:
Fixed few other editorial issues

Fixed few other editorial issues based on suggestions in build

v-thepet committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fixlinks

PRMerger19 committed:
Merge pull request #21738 from MarkusVi/ca05

ca05

mimig1 committed:
Cosmos DB: Obscured details from screencap.

sethmanheim committed:
Adding article about migrating from ACS to SAS

sethmanheim committed:
Fixing metadata

sethmanheim committed:
Fixing links

Commit:
ca06

v-shils committed:
Merge pull request #21740 from mimig1/09072017l2

Cosmos DB: Obscured details from screencap.

PRMerger10 committed:
Merge pull request #21617 from anirudhcavale/patch-2

Reflecting most up to date changes

v-thepet committed:
fix breaks

PRMerger18 committed:
Merge pull request #21743 from MarkusVi/ca06

ca06

TerryLanfear committed:
one final fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sec-acs

v-thepet committed:
Merge pull request #21741 from sethmanheim/wk-relay

Adding Relay article about migrating from ACS to SAS

v-thepet committed:
Merge pull request #21728 from sethmanheim/wk9-7

Adding Service Bus article about migrating from ACS to SAS

Commit:
ca07

Commit:
Merge pull request #21747 from v-thepet/fixlinks

production bug: Fixlinks

robinsh committed:
Make changes req by PR Reviewer

sethmanheim committed:
Removing deprecation date

PRMerger10 committed:
Merge pull request #21749 from MarkusVi/ca07

ca07

PRMerger20 committed:
Merge pull request #21751 from robinsh/robin-0907

Make changes req by PR Reviewer

PRMerger14 committed:
Merge pull request #21750 from sethmanheim/sas-fix

Removing deprecation date

Commit:
ca08

ahmedelnably committed:
editing preview pricing question

Commit:
missed instances based off text search

cwatson-cat committed:
Updated master redirection file to change doc id to true for moved storage files.

Ja-Dunn committed:
Merge pull request #21648 from TerryLanfear/sec-acs

update to FAQ

Ja-Dunn committed:
Merge pull request #21757 from MarkusVi/ca08

ca08

ahmedelnably committed:
reverting .gitignore file

Commit:
fix incorrect cmdlet names

ahmedelnably committed:
change

ahmedelnably committed:
Revert "change"

This reverts commit 5a4b2da91d2bd6ca6ed527dab4aabfcf2421d4c0.

ahmedelnably committed:
Revert "reverting .gitignore file"

This reverts commit 9690ff1afba1e3751d3d70175071e3611fc3a363.

ahmedelnably committed:
Revert "editing preview pricing question"

This reverts commit 620e8e6c9dfbb4198ce876da3feb3405a3dec46d.

chaosrealm committed:
Updated preview API version

ahmedelnably committed:
fix preview pricing

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-dev-guide-updates

carolinacmoravia committed:
P: Broken metadata due to extra spaces

Ja-Dunn committed:
Merge pull request #20914 from dksimpson/dean-1708

Dean 1708

nitinme committed:
Incorporating dev guide updates for these articles

Commit:
Merge pull request #21761 from chaosrealm/patch-1

Updated preview API version

PRMerger19 committed:
Merge pull request #21758 from msebolt/iot-hub-amqp-update-revised

missed instances based off text search

PRMerger12 committed:
Merge pull request #21762 from carolinacmoravia/patch-381

P: Broken metadata due to extra spaces

Ja-Dunn committed:
Merge pull request #21759 from cwatson-cat/9-7-storage-mstrdr-docidupd

Updated master redirection file to change doc id=true for moved storage files

nitinme committed:
Acrolinx updates

v-thepet committed:
Merge pull request #21602 from dksimpson/dean-315

task 315 - add validation status table

ShawnJackson committed:
Update notification-hubs-aspnet-cross-platform-notification.md

mahesh-unnikrishnan committed:
Remove deprecated classic portal instructions to setup AAD-DS. Redirect to
Azure portal instructions.

Juliako committed:
Update media-services-face-redaction.md

PRMerger20 committed:
Merge pull request #21767 from ahmedelnably/master

Fixing preview pricing question

v-thepet committed:
Merge pull request #19335 from vigneshcmsft/patch-1

Update stream-analytics-out-of-order-and-late-events.md

Juliako committed:
Update media-services-redactor-walkthrough.md

Commit:
fixing Acrolinx issues

Commit:
minor edits

ShawnJackson committed:
Merge pull request #20094 from itechedit/notification-hubs-aspnet-cross-platform-notification

edit pass: notification-hubs-aspnet-cross-platform-notification

ShawnJackson committed:
Merge pull request #20253 from itechedit/notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

edit pass: notification-hubs-windows-notification-dotnet-push-xplat-segmented-wns

Ja-Dunn committed:
Merge pull request #21760 from sdwheeler/sdw-fixlinktext

fix incorrect cmdlet names

Commit:
Merge pull request #21774 from msebolt/freshness-reviews

freshness review

ShawnJackson committed:
Merge pull request #20573 from itechedit/notification-hubs-windows-store-dotnet-get-started-wns-push-notification

edit pass: notification-hubs-windows-store-dotnet-get-started-wns-push-notification

nitinme committed:
Fixing PR comment

Commit:
b-series

mimig1 committed:
Cosmos DB: Bullet about support for try it accts

ShawnJackson committed:
Update notification-hubs-aspnet-backend-notifyusers.md

Commit:
fixing toc

steved0x committed:
Update link for Azure Storage explorer 

Per @cawa

Commit:
fixing

PRMerger18 committed:
Merge pull request #21776 from MicrosoftDocs/mimig1-patch-2

Cosmos DB: Bullet about support for try it accts

ShawnJackson committed:
Merge pull request #20602 from itechedit/includes-notification-hubs-aspnet-backend-notifyusers

edit pass: includes-notification-hubs-aspnet-backend-notifyusers

v-thepet committed:
Merge pull request #21766 from nitinme/nitinme-hdi-dev-guide-updates

Incorporating dev guide updates # 1

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
ca10

v-thepet committed:
fix

ShawnJackson committed:
Merge pull request #21613 from paulathurman/azure-security-threat-modeling-tool-feature-overview

edit pass: azure-security-threat-modeling-tool-feature-overview

v-thepet committed:
Merge pull request #21780 from steved0x/patch-1

Update link for Azure Storage explorer

PRMerger16 committed:
Merge pull request #21778 from cynthn/b-series

b-series

mahesh-unnikrishnan committed:
retire files as per PR review feedback.

mahesh-unnikrishnan committed:
Merge branch 'master' into master

PRMerger13 committed:
Merge pull request #21770 from Juliako/patch-16

Update media-services-face-redaction.md

Commit:
Merge pull request #21771 from Juliako/patch-17

Update media-services-redactor-walkthrough.md

Commit:
ca10

Commit:
ca11

PRMerger16 committed:
Merge pull request #21791 from v-thepet/fixlinks

production bug: fix

PRMerger18 committed:
Merge pull request #21790 from MarkusVi/ca10

ca10

PRMerger13 committed:
Merge pull request #21800 from MarkusVi/ca11

ca11

v-thepet committed:
Merge pull request #21772 from mahesh-unnikrishnan/master

Remove deprecated classic portal instructions to setup AAD-DS.

v-alje committed:
Merge pull request #21788 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
